_includes/configuration/language.md

```diff
@@ -4,12 +4,12 @@ This option specifies the language used for the {{site.productname}} user interf
 
 Before changing the language option, ensure that the language pack is available to the {{site.productname}} instance. {{site.companyname}} provides two collections of language packs:
 
-- _Premium_ language packs - Professionally localized language packs provided on {{site.cloudname}} and bundled with premium self-hosted bundles.
+- _Enterprise_ language packs - Professionally localized language packs provided on {{site.cloudname}} and bundled with enterprise self-hosted bundles.
 - _Community_ language packs - Localizations provided by {{site.productname}} users through Transifex, which need to be downloaded prior to use, from [the {{site. companyname}} Downloads Page - Language Packages]({{site.gettiny}}/language-packages/).
 
 For information on:
 
-- Using the premium language packs, see: [Using the premium language packs](#usingthepremiumlanguagepacks).
+- Using the enterprise language packs, see: [Using the enterprise language packs](#usingtheenterpriselanguagepacks).
```

Comment:
This change breaks the link. You need to update the referencing Id(s) in [localization.html](https://github.com/tinymce/tinymce-docs/blob/hotfix/5/DOC-1540_2/build/site/tinymce/5/configure/localization.html#L1468)

Reply:
@tiny-ben-tran the id/title used to generate that is below on line 32? Did I miss something here otherwise?

Also your link doesn't work and is to `build` which is a temporary build directory, not source code.

Reply:
You are right. The localization.html was generated. VScode github deceived me